src/main/java/net/minecraftforge/client/ForgeHooksClient.java

```diff
@@ -114,12 +115,7 @@
 import java.io.IOException;
 import java.nio.Buffer;
 import java.nio.ByteBuffer;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Objects;
-import java.util.Optional;
-import java.util.Random;
-import java.util.Set;
+import java.util.*;
```

Comment:
Replace the wildcard import with the classes needed. I imagine there's a setting in your IDE which handles that.